src/crs.jl

```diff
@@ -137,3 +137,53 @@ Base.convert(::Type{CRS}, crs::GFT.CoordinateReferenceSystemFormat) = CRS(crs)
 
 # Maybe enable later, based on https://github.com/JuliaGeo/GeoFormatTypes.jl/issues/21
 # Base.convert(T::Type{<:GFT.CoordinateReferenceSystemFormat}, crs::GFT.CoordinateReferenceSystemFormat) = T(CRS(crs))
+
+
+
+
+"""
+    identify(obj; authority = nothing, options = nothing)
+
+Returns the a list of matching reference CRS or nothing, and the confidence values (0-100) 
+or nothing. This is a high level interface to proj_identify.
+
+Inputs:
+   obj -- Object of type CRS. Must not be NULL
+   authority [nothing] -- Authority name, or NULL for all authorities (e.g. "EPSG")
```

Comment:
In general I'd like to stay close to the PROJ API unless there is good reason to deviate. Shall we just keep calling it `auth_name`?

Reply:
sounds good.. updated